src/sentry/api/serializers/snuba.py

```diff
@@ -342,4 +346,8 @@ def serialize(self, result, column="count", order=None, allow_partial_buckets=Fa
         elif "order" in result.data:
             res["order"] = result.data["order"]
 
+        if hasattr(result, "start") and hasattr(result, "end"):
+            res["start"] = result.start
+            res["end"] = result.end
```

Comment:
Nit: should this be set to the epoch instead of the string to be consistent with the rest of the response? Not sure how that works together with ECharts though.

Reply:
Updated to epoch time, works with ECharts during my testing!